pkg/component/networking/istio/test_charts/ingress_vpn_envoy_filter.yaml

```diff
@@ -22,76 +22,35 @@ spec:
           route_config:
             virtual_hosts:
             - domains:
-              - api.*
+              - "*"
               name: reversed-vpn
               routes:
               - match:
                   connect_matcher: {}
+                  headers:
```

Comment:
Can you drop the commits of https://github.com/gardener/gardener/pull/11328 from this PR?

Reply:
done 😄